app/features-json/proto/project_summary_proto.rb

```diff
@@ -0,0 +1,21 @@
+require_relative "../../shared/models/project"
+
+module FastlaneCI
+  # Proto to expose the basic info about a project.
+  class ProjectSummaryProto
+    class << self
+      def object_class
+        FastlaneCI::Project
+      end
+
+      def proto_from(object:)
```

Comment:
I don't know that `proto` is the right word for this. I understand you're thinking about protobufs, but maybe there is some other word we can use that is more familiar to devs who don't know protobuf stuff.

Reply:
I 100% agree. I don't know what the correct term is. But this is something much simpler and might eventually get replaced by something more like a proto.

Reply:
If this is used solely for the view, I might learn towards `ViewData`

Reply:
`ViewModel`? 😇